tests/functional/fixtures/happy_path_project/macros/macro_stuff.sql

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
super nit: looks like a lot of these files have an extra empty line at the top of the file, could we clean that up?